src/main/java/org/broadinstitute/hellbender/utils/test/ArgumentsBuilder.java

```diff
@@ -17,7 +17,10 @@
 public final class ArgumentsBuilder {
     private final List<String> args= new ArrayList<>();
 
-    public ArgumentsBuilder(){
+    public ArgumentsBuilder(){}
+
+    public static ArgumentsBuilder init(){
```

Comment:
what's the purpose of this method? is it better/different than just `new ArgumentsBuilder()` ?